demo/demo-points.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>Leaflet Map Panes Example</title>
+	<title>Leaflet.VectorGrid Points Example</title>

```

Comment:
Well actually... https://github.com/Leaflet/Leaflet.VectorGrid/blob/master/demo/demo-vectortiles.html#L4

...and...

https://github.com/Leaflet/Leaflet.VectorGrid/blob/master/demo/map-panes-example.html

:stuck_out_tongue_winking_eye: